### PR TITLE
Tags index design update

### PR DESCRIPTION
**Pull request by ludwiczakpawel**
## What type of PR is this? (check all applicable)

- [x] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

This is a little layout and design update to our `/tags` page. It should look slightly bit better on production comparing with my screenshot since I have very little data locally and no logos :C. Definitely not a final state, but I think it's good-enough first step.

## QA Instructions, Screenshots, Recordings

Before

image

After

image

## Added tests?

- [ ] yes
- [x] no, because they aren't needed
- [ ] no, because I need help

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [x] no documentation needed

### CODE REVIEW

File: app/views/tags/index.html.erb [hunk 1]
```diff
@@ -19,25 +19,52 @@
...
+
+  .tag-card:not(:hover) .crayons-btn--secondary.follow-action-button {
+    background: var(--button-secondary-bg) !important;
+    color: var(--button-secodnary-color) !important;
+  }
+</style>
+
+<div class="crayons-layout crayons-layout--1-col">
+  <header class="px-2 m:px-0">
     <h1>Top 100 Tags</h1>
```
Comment (rhymes) [inline app/views/tags/index.html.erb:36]: Just thought that there's no guarantee that a website has 100 tags :D 
https://letsbuild.gg/tags has 81
https://www.thismmalife.com/tags has 78.

I think we should drop the `100` from the title

File: app/views/tags/index.html.erb [hunk 2]
```diff
@@ -19,25 +19,52 @@
...
+        <h3 class="crayons-tag crayons-tag--l mb-2">
+          <a href="/t/<%= tag.name %>" class="crayons-link">
+            <span class="crayons-tag__prefix">#</span><%= tag.name %>
+          </a>
+        </h3>
+        <% if tag.short_summary.present? %>
+          <p class="mb-2 truncate-at-3"><%= sanitize tag.short_summary %></p>
+        <% end %>
+        <p class="mb-3 fs-s color-base-50">
+          <%= pluralize @num_published_articles, "post" %> published
```
Comment (rhymes) [inline app/views/tags/index.html.erb:52]: I don't think this is going to work, it comes from a different controller.
You can use `tag.taggings_count`

File: app/views/tags/index.html.erb (hunk 1 above)
Comment (ludwiczakpawel) [reply app/views/tags/index.html.erb:36]: Very good point. I'll fix it. Just gonna have "Top Tags" for now and eventually maybe "Tags"

File: app/views/tags/index.html.erb [hunk 3]
```diff
@@ -19,25 +19,52 @@
   <meta name="twitter:image:src" content="<%= SiteConfig.main_social_image %>">
 <% end %>
 
-<div class="user-profile-header tag-header bg-transparent">
-  <div class="tag-or-query-header-container">
+<style>
```
Comment (nickytonline) [inline app/views/tags/index.html.erb:22]: Any reason for the `<style />` tag addition? Is this inline critical CSS?

Comment (ludwiczakpawel) [reply app/views/tags/index.html.erb:22]: Not critical, but useful :D I've done little "redesign" since yesterday, so we should not need it anymore.

**Comment by ludwiczakpawel** [comment]
@rhymes I have indeed changed "branding" strategy a little. go ahead and take it for another ride.

**Comment by ludwiczakpawel** [comment]
@rhymes you're right, i've just bumped the contrast ratio - should be good now.

**Pull request merged**